src/crab/bitset_domain.hpp

```diff
@@ -76,4 +76,15 @@ class bitset_domain_t final {
     }
 
     friend std::ostream& operator<<(std::ostream& o, const bitset_domain_t& array);
+
+    bool all_num(int lb, int ub) {
```

Comment:
Assuming the lower bound is inclusive and the upper bound is exclusive, the code looks right, but a comment to the effect would help.
```suggestion
    // Test whether all values in the range [lb,ub) are numerical.
    bool all_num(int lb, int ub) {
```